diff --git a/mongoose.c b/mongoose.c
index 7e55896..17ff03a 100644
--- a/mongoose.c
+++ b/mongoose.c
@@ -1079,14 +1079,14 @@ char *cs_md5(char buf[33], ...) {
 #define MBUF_FREE free
 #endif
 
-void mbuf_init(struct mbuf *mbuf, size_t initial_size) WEAK;
+//void mbuf_init(struct mbuf *mbuf, size_t initial_size) WEAK;
 void mbuf_init(struct mbuf *mbuf, size_t initial_size) {
   mbuf->len = mbuf->size = 0;
   mbuf->buf = NULL;
   mbuf_resize(mbuf, initial_size);
 }
 
-void mbuf_free(struct mbuf *mbuf) WEAK;
+//void mbuf_free(struct mbuf *mbuf) WEAK;
 void mbuf_free(struct mbuf *mbuf) {
   if (mbuf->buf != NULL) {
     MBUF_FREE(mbuf->buf);
